src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -296,7 +296,7 @@ public ItemStack removeItemNeeded(@NotNull final ItemStack stack)
 
         for (@NotNull final ItemStack neededItem : itemsNeeded)
         {
-            if ((stack.getItem().isDamageable() && stack.getItem() == neededItem.getItem()) || stack.isItemEqual(neededItem))
+            if (stack.isItemEqualIgnoreDurability(neededItem))
             {
                 //todo make this sofisticated as soon as material handling has been implemented.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)